src/ucp/core/ucp_context.c

```diff
@@ -247,15 +248,15 @@ UCS_CONFIG_REGISTER_TABLE(ucp_config_table, "UCP context", NULL, ucp_config_t)
 
 
 static ucp_tl_alias_t ucp_tl_aliases[] = {
-  { "sm",    { "mm", "knem", "cma", "rdmacm", NULL } },
-  { "shm",   { "mm", "knem", "cma", "rdmacm", NULL } },
-  { "ib",    { "rc", "ud", "rc_mlx5", "ud_mlx5", "dc_mlx5", "rdmacm", NULL } },
-  { "ud",    { "ud", "rdmacm", NULL } },
-  { "ud_x",  { "ud_mlx5", "rdmacm", NULL } },
-  { "rc",    { "rc", "ud:aux", "rdmacm", NULL } },
-  { "rc_x",  { "rc_mlx5", "ud_mlx5:aux", "rdmacm", NULL } },
-  { "dc",    { "dc_mlx5", "rdmacm", NULL } },
-  { "dc_x",  { "dc_mlx5", "rdmacm", NULL } },
+  { "sm",    { "mm", "knem", "cma", "rdmacm", "sockcm", NULL } },
+  { "shm",   { "mm", "knem", "cma", "rdmacm", "sockcm", NULL } },
+  { "ib",    { "rc", "ud", "rc_mlx5", "ud_mlx5", "dc_mlx5", "rdmacm", "sockcm", NULL } },
+  { "ud",    { "ud", "rdmacm", "sockcm", NULL } },
```

Comment:
why adding sockcm to IB transports? Isn't it enough to add it to TCP only?

Reply:
Sorry needed this for my experiments. Removing this now.

Reply:
@yosefe It's ok to still have this for `sm/shm` right?